src/CATs.js

```diff
@@ -91,9 +91,9 @@ Color.defineCAT({
 Color.defineCAT({
 	id: "CAT16",
 	toCone_M: [
-		[  0.401288,  0.650173, -0.051461 ],
-		[ -0.250268,  1.204414,  0.045854 ],
-		[ -0.002079,  0.048952,  0.953127 ]
+		[  0.4012880,  0.6501730, -0.0514610 ],
```

Comment:
In general I agree about making clear the number of significant figures and thus, not dropping trailing zeroes.